tc_pdf/handlers/pdf.py

```diff
@@ -115,11 +116,11 @@ def create_preview(self, url, resolution=200):
                             i.background_color = Color('white')  # Set white background.
                             i.alpha_channel = 'remove'  # Remove transparency and replace with bg.
                             i.save(file=out_io)
-                            raise gen.Return(out_io.getvalue())
+                            return out_io.getvalue()
 
                 except WandException as e:
                     logger.exception('[PDFHander.create_preview] %s', e)
```

Comment:
```suggestion
                    logger.exception('[PDFHandler.create_preview] %s', e)
```

Reply:
I'm not sure if this change is in the scope of this PR, because the typo wasn't introduced by it, but anyways it's better to fix it.